torch/utils/benchmark/utils/timer.py

```diff
@@ -56,7 +155,8 @@ def __init__(
             num_threads=num_threads,
         )
 
-    def timeit(self, number=1000000):
+    def timeit(self, number: int = 1000000) -> common.Measurement:
+        """Mirrors the semantics of timeit.Timer.timeit()."""
```

Comment:
1 sentence description (times `number` executions of the main statement) + a link to timeit.Timer.timeit()?

Reply:
Done.